numpy/linalg/lapack_lite/f2c.h

```diff
@@ -214,4 +214,108 @@ typedef doublereal E_f;	/* real function with -R not specified */
 #undef unix
 #undef vax
 #endif
+
+#if defined(KR_headers)
+VOID z_sqrt(r,z)doublecomplex *r,*z;
+VOID c_sqrt(r,z)complex *r,*z;
+extern integer *b;
+extern integer *b;
+VOID c_div(c,a,b)complex *a,*b,*c;
+void sig_die();
+#endif
+#if !(defined(KR_headers))
+void sig_die(char *,int);
+#endif
+#if defined(KR_headers)
+void sig_die();
+#endif
+#if !(defined(KR_headers))
+void sig_die(char *,int);
+#endif
+#if defined(KR_headers)
+double z_abs(z)doublecomplex *z;
+double c_abs(z)complex *z;
+extern ftnlen la,lb;
+extern ftnlen la,lb;
+VOID s_copy(a,b,la,lb)register char *a,*b;
+integer s_cmp(a0,b0,la,lb)char *a0,*b0;
+extern ftnlen rnp[],*np,ll;
+s_cat(lp,rpp,rnp,np,ll)char *lp,*rpp[];
+#endif
+#if defined(KR_headers) && !defined(NO_OVERWRITE)
+void exit_();
+void free();
+char *F77_aloc();
+#endif
+#if !(defined(KR_headers)) && !defined(NO_OVERWRITE)
+char *F77_aloc(ftnlen,char *);
+#endif
+#if defined(KR_headers)
+VOID pow_ci(p,a,b)complex *p,*a;
+#endif
+#if !(defined(KR_headers))
+void z_div(doublecomplex *,doublecomplex *,doublecomplex *);
+#endif
+#if defined(KR_headers)
+VOID z_div(c,a,b)doublecomplex *a,*b,*c;
+VOID pow_zi(p,a,b)doublecomplex *p,*a;
+#endif
+#if !(defined(KR_headers))
+void pow_zi(doublecomplex *,doublecomplex *,integer *);
+#endif
+#if defined(KR_headers)
+extern integer *bp;
+extern integer *bp;
+double pow_di(ap,bp)doublereal *ap;
+double pow_ri(ap,bp)real *ap;
+double pow_dd(ap,bp)doublereal *ap,*bp;
+double pow();
+integer i_nint(x)real *x;
+integer i_dnnt(x)doublereal *x;
+double floor();
+double floor();
+double d_sign(a,b)doublereal *a,*b;
+double r_sign(a,b)real *a,*b;
+double d_lg10(x)doublereal *x;
+double log();
+double r_lg10(x)real *x;
+float logf();
+double d_imag(z)doublecomplex *z;
+float r_imag(z)complex *z;
+d_cnjg(r,z)doublecomplex *r,*z;
+r_cnjg(r,z)complex *r,*z;
+double sqrt();
+double f__cabs(real,imag)double real,imag;
+#endif
+#if !(defined(KR_headers))
+double f__cabs(double,double);
+#endif
+#if defined(KR_headers)
+double sqrt(),f__cabs();
+#endif
+#if !(defined(KR_headers))
+double f__cabs(double,double);
+#endif
+#if defined(KR_headers)
+double f__cabs();
+float sqrtf();
+double f__cabsf(real,imag)float real,imag;
+#endif
+#if !(defined(KR_headers))
+double f__cabsf(float,float);
+#endif
+#if defined(KR_headers)
+float sqrtf(),f__cabsf();
+#endif
+#if !(defined(KR_headers))
+double f__cabsf(float,float);
+#endif
+#if defined(KR_headers)
+double f__cabsf();
+#endif
```

Comment:
`#if defined(KR_headers)` seems to be doing the wrong thing. We don't actually need to support this use case anyway - it just happens that `f2c.c` is a concatenation of files from netlib, which are otherwise untouched.

Reply:
We can't remove this from the header file unless we remove it from the C file as well.

Reply:
Ah, just looked it up - what you have is the correct way to declare K&R prototypes